IUPAC_SMILES+.asciidoc

```diff
@@ -1125,38 +1170,36 @@ Very few SMILES systems actually implement the rules for `SP`, `TB` or `OH` chir
 == Parsing Termination
 
 A SMILES string is terminated by a whitespace terminator character (space, tab, newline,
-carriage-return), or by the end of the string.
+carriage-return), or by the end of the string. As a result, and leading space in a SMILES string
```

Comment:
I believe "As a result, and leading space" should read "As a result, any leading space".